code/game/objects/items/devices/scanners/health_analyzer.dm

```diff
@@ -591,6 +570,24 @@
 		return
 
 	diseasescan(user, patient, src)
+//Checks the individual for any diseases that are visible to the scanner, and displays the diseases in the attacked to the attacker.
+/proc/diseasescan(mob/user, mob/living/carbon/patient, obj/item/healthanalyzer/wound/scanner)
+	if(!istype(patient) || user.incapacitated())
+		return
+
+	var/render_list = ""
```

Comment:
you should make this a /list called just render

Reply:
This is how all of the health analyzers are, I'm sure how in scope this is.

Reply:
You don't have to, it would improve the code quality significantly and should be reasonably trivial